src/qibocal/protocols/characterization/utils.py

```diff
@@ -27,11 +28,31 @@
 LEGEND_FONT_SIZE = 20
 TITLE_SIZE = 25
 EXTREME_CHI = 1e4
+KB = constants.physical_constants["Boltzmann constant in Hz/K"][0]
+HBAR = constants.hbar
 """Chi2 output when errors list contains zero elements"""
 COLORBAND = "rgba(0,100,80,0.2)"
 COLORBAND_LINE = "rgba(255,255,255,0)"
 
 
+def effective_qubit_temperature(
+    prob_0: np.array, prob_1: np.array, qubit_frequency: float
+):
+    """Calculates the qubit effective temperature.
+
+    The formula used in this one:
+
+    kB Teff = - hbar qubit_freq / ln(prob_1/prob_0)
+
+    """
+    try:
+        temp = -HBAR * qubit_frequency / np.log(prob_1 / prob_0) / KB
```

Comment:
```suggestion
        temp = -HBAR * qubit_frequency / (np.log(prob_1 / prob_0) * KB)
```
Could you group all divisions?

Reply:
Sure